keyboards/gray_studio/think65/hotswap/keymaps/via/think65-hotswap.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Aren't these meant to go to the VIA repo?

Reply:
Yep, and I plan to submit a PR for them soon, but I also think they're worth keeping in the QMK keymap dir too just in case. I'm ultimately ambivalent though, so let me know if you'd rather they were removed.

Reply:
Resolved by 8a2cb6c